### ISSUE TITLE
individual words are broken up

### ISSUE THREAD

**Issue by phoebeireland** [gouravkhunger/nextjs-birthday-wish#22]
on the second page (once you've clicked the go button), the words on the second page are broken up if the screen gets smaller.
someone else can take this request
![textcutoff](https://user-images.githubusercontent.com/64537388/135916904-6c082589-4bd4-43e9-9e73-5e209c989a11.jpg)
.

**Comment by gouravkhunger** [comment]
Yeah I was gonna open an issue on that. This feature was implemented by @PhattOZ. Maybe if he knows some fix he can try 🤔

**Comment by PhattOZ** [comment]
I’ll look into it. Thank you for pointing it out! 💯

**Comment by gouravkhunger** [comment]
Yeah sure, I think we can do something like giving each word a span wrapper and keep the three span elements are inside a div as flex and set up animation on singular elements and that should sync with each other to produce a similar effect.

Should I assign this to you @PhattOZ ?

**Comment by PhattOZ** [comment]
Sure! I’ll take care of it
